fuzz/fuzz_targets/fuzz_test.rs

```diff
@@ -26,8 +28,11 @@ enum ArgType {
     // Add any other types as needed
 }
 
+static CMD_PATH: &str = "/usr/bin/test";
+
 fn run_gnu_test(args: &[OsString]) -> Result<(String, i32), std::io::Error> {
-    let mut command = Command::new("test");
+    let mut command = Command::new(CMD_PATH);
```

Comment:
Is this deliberately done differently than in `fuzz_expr.rs`? There it is done in the following way:
```rust
static CMD_PATH: &str = "expr";

fn run_gnu_expr(args: &[OsString]) -> Result<(String, i32), std::io::Error> {
    is_gnu_cmd(CMD_PATH)?; // Check if it's a GNU implementation

    let mut command = Command::new(CMD_PATH);

```

Reply:
/usr/bin/ is an artifact, i will remove it

and yeah, test --version doesn't return anything :(
so, dunno how to verify that it is the GNU impl

Reply:
In the [docs](https://www.gnu.org/software/coreutils/manual/html_node/test-invocation.html#test-invocation) they write to use `[`:

> To get help and version information, invoke the commands ‘[ --help’ and ‘[ --version’